src/orion/benchmark/task/quadratics.py

```diff
@@ -0,0 +1,115 @@
+""" Defines the Quadratics task, as described in section 4.2 of the ABLR paper.
+
+[1] [Scalable HyperParameter Transfer Learning](
+    https://papers.nips.cc/paper/7917-scalable-hyperparameter-transfer-learning)
+"""
+from dataclasses import dataclass
+from logging import getLogger as get_logger
+from typing import Dict, Optional, List
+
+import numpy as np
+from orion.benchmark.task.base import BaseTask
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class QuadraticsTaskHParams:
+    """ Hyper-parameters of the Quadratics task. """
+
+    x0: float
+    x1: float
+    x2: float
+
+
+@dataclass
+class QuadraticsTaskHparamsWithContext(QuadraticsTaskHParams):
+    """ Hyper-parameters of the Quadratics task that also include the context vector.
+
+    The context vector in this case are the coefficients of the quadratic.
+    
+    NOTE: This adds these entries to the space, but they are ignored when evaluating a given point
+    (they aren't used by the `__call__` method of QuadraticsTask).
+    """
+
+    a2: float
+    a1: float
+    a0: float
+
+
+class QuadraticsTask(BaseTask):
+    """ Simple task consisting of a quadratic with three coefficients, as described in ABLR.
+    
+    NOTE: In the paper, section 4.2, the function is defined over R^3, but here
+    the bounds are limited to [-100,100] for now since otherwise y can be enormous.
```

Comment:
```suggestion
   Notes
   -----
   In the paper, section 4.2, the function is defined over R^3, but here
   the bounds are limited to [-100,100] for now since otherwise y can be enormous.
```